src/Raven.Client/Documents/Operations/ConnectionStrings/PutConnectionStringOperation.cs

```diff
@@ -68,6 +68,9 @@ public override void SetResponse(JsonOperationContext context, BlittableJsonRead
 
     public class PutConnectionStringResult
     {
+        [Obsolete("Not used to get operation result. Use RaftCommandIndex instead")]
```

Comment:
`[Obsolete("PutConnectionStringResult.ETag is not supported anymore. Will be removed in next major version of the product. Please use PutConnectionStringResult.RaftCommandIndex instead.")]`

Reply:
done